aim/storage/encoding/encoding_native.cpp

```diff
@@ -0,0 +1,5509 @@
+/* Generated by Cython 0.29.24 */
```

Comment:
The auto-generated sources should not appear in the git repo.

Reply:
yea, will remove them, I have moved away from this solution alltogether.
will push the Pytorch Lightning  sphinx based one soon

Reply:
@mahnerak removed the autogenerated files